main.go

```diff
@@ -0,0 +1,81 @@
+// Mathis Van Eetvelde
+// Ghent 2021
+
+package main
```

Comment:
Please run go fmt and goreportcard locally

Reply:
```
Grade: A+ (94.1%)
Files: 1
Issues: 1
gofmt: 100%
go_vet: 100%
gocyclo: 100%
golint: 100%
ineffassign: 100%
license: 0%
misspell: 100%
```
Check slack for the license problem

Reply:
Replied on slack. Please check there.